GameFighter/Models/Person.cs

```diff
@@ -12,9 +12,23 @@ public abstract class Person
 
         public bool IsAlive => Health > 0;
 
-        public virtual void GetAttack(Warrior warrior)
+
+        public virtual int GetAttack(Warrior warrior)
         {
+            int beforeFightHealth = Health;
+            
             Health -= warrior.Attack;
+
+            if (Health > 0)
```

Comment:
what is the purpose of this part? I'm not sure that you need to handle this case. But if you think that it's okay, you can leave it. In this case use ternary operator

Reply:
I think, that vampires cannot heal from dead fighters